CHANGELOG.md

```diff
@@ -2,78 +2,83 @@
 
 ## [Unreleased](https://github.com/confio/poe-contracts/tree/HEAD)
 
-[Full Changelog](https://github.com/confio/poe-contracts/compare/v0.6.0-rc2...HEAD)
+[Full Changelog](https://github.com/confio/poe-contracts/compare/v0.6.0...HEAD)
 
-## [v0.6.0-rc2](https://github.com/confio/poe-contracts/tree/v0.6.0-rc2) (2022-02-03)
+## [v0.6.0](https://github.com/confio/poe-contracts/tree/v0.6.0) (2022-02-15)
+
+[Full Changelog](https://github.com/confio/poe-contracts/compare/v0.6.0-rc2...v0.6.0)
 
 **Breaking changes:**
 
-- Use specilaized tg3 version of voting API for tgrade contracts [\#85](https://github.com/confio/poe-contracts/issues/85)
-- Valset config contract names [\#96](https://github.com/confio/poe-contracts/pull/96) ([maurolacy](https://github.com/maurolacy))
-- tg3: Common voting interfaces for tgrade [\#93](https://github.com/confio/poe-contracts/pull/93) ([hashedone](https://github.com/hashedone))
-- Rename API to points and rewards [\#50](https://github.com/confio/poe-contracts/pull/50) ([ethanfrey](https://github.com/ethanfrey))
+- Rename `Tg4Contract::total_weight` to `Tg4Contract::total_points` in `packages/tg4`.
+
+- Some `Response` `attributes` renaming in `tg4-engagement`:
+  - "distribute_tokens" -> "distribute_rewards"
+  - "withdraw_tokens" -> "withdraw_rewards"
+  - "token" -> "rewards"
+  -  "weight" -> "points"
 
 **Closed issues:**
 
-- Valset: add `ListJailedValidators` query [\#87](https://github.com/confio/poe-contracts/issues/87)
-- Add `list_votes_by_voter` query to `voting_contract` [\#78](https://github.com/confio/poe-contracts/issues/78)
-- Increase max limit [\#76](https://github.com/confio/poe-contracts/issues/76)
-- Missing items found when updating Go code [\#75](https://github.com/confio/poe-contracts/issues/75)
-- Tools to help build API Migration Guide [\#72](https://github.com/confio/poe-contracts/issues/72)
-- \[tgrade-valset\] Metadata issues [\#66](https://github.com/confio/poe-contracts/issues/66)
-- Ensure validator is online before adding to the active set [\#36](https://github.com/confio/poe-contracts/issues/36)
+- Refactor local names to match new naming rules [\#86](https://github.com/confio/poe-contracts/issues/86)
```

Comment:
So this issue is possibly breaking, as you wrote in aligned PR: https://github.com/confio/poe-contracts/pull/102#discussion_r806691536

Although I think you mentioned it anyway "manually"